[IMP] ecommerce: updated variants page (#3299)
odoo/documentation: content/applications/websites/ecommerce/managing_products/variants.rst

================
Product variants
================

Product variants are variations such as color, material, etc. of a same product. These variations
can differ in price and availability from the product. Product variants can either be
:doc:`created <../../../sales/sales/products_prices/products/variants>` or
:doc:`imported <../../../sales/sales/products_prices/products/import>`.

.. note::

>    :guilabel:`Product Variants` has to be enabled under :menuselection:`Configuration --> Settings`,
>    in the :guilabel:`Shop - Products` section.

Reviewer: ```suggestion
   :guilabel:`Product Variants` have to be enabled under :menuselection:`Configuration --> Settings`,
   in the :guilabel:`Shop - Products` section.
```

I know it's "the option" that has to be enabled, but it sounds weird when the words before the verb are in the plural form

.. seealso::
   - :doc:`../../../sales/sales/products_prices/products/variants`
   - :doc:`../../../sales/sales/products_prices/products/import`

Product configurator
====================

> In the context of eCommerce, adding attributes and values to a product template allows for the
> addition of a **product configurator** on the product page. The customer is able to configure and
> select the product variant of their choice; or in the case of multiple attributes, combine those to
> create a specific variant.

Reviewer: ```suggestion
In the context of eCommerce, adding attributes and values to a product template allows for adding a **product configurator** on the product page. Customers are able to configure and
select the product variant of their choice; or in the case of multiple attributes, combine those to
create a specific variant.
```
I would change it to a plural as "the customer" (sg) --> of "his/her" choice

Author: I changed it up a bit, thanks for the suggestion! FYI, "their" is the neutral form (and therefore preferred), and works for both Sg. and Pl. :)

.. image:: variants/variants-configurator.png
   :align: center
   :alt: Variants configurator

The **display type** of each attribute used in the product configurator can be edited through the
**website builder** by clicking :menuselection:`Edit --> Customize` on the product page, and then
clicking on one of the attributes. You can then select between four options:

- :guilabel:`Radio`
- :guilabel:`Pills`
- :guilabel:`Select`
- :guilabel:`Color`

.. image:: variants/variants-display-type.png
   :align: center
   :alt: Display type options for attributes

.. _variants-configurator-backend:

Alternatively, the **display type** can also be edited through the **back end**
(:menuselection:`Product template --> Attributes`), by clicking :guilabel:`Configure` on the **value
line**.

.. tip::

>    Specific combinations of values can be excluded from the product configurator. The customers are
>    then not able to select the excluded combination of values. To do so, access the
>    :ref:`Configure <variants-configurator-backend>` menu, select a **value**, and fill in the
>    :guilabel:`Exclude for` line.

Reviewer: ```suggestion
   Specific combinations of values can be excluded from the product configurator. This way, customers are
  unable to select the excluded combination of values. To do so, access the
   :ref:`Configure <variants-configurator-backend>` menu, select a **value**, and fill in the
   :guilabel:`Exclude for` line.
```

Product specifications
======================

Values used for each attribute are displayed as a **specification list** at the bottom of the
product page. To be visible, the **specification list** must first be enabled on the product page by
going to :menuselection:`Edit --> Customize` and selecting a placement for the field
:guilabel:`Specification`.

.. image:: variants/variants-specifications.png
   :align: center
   :alt: Specifications list on the product page

.. tip::
   The product **specification list** can also be used on products without variants. For that, make
   sure to have no values combination. Products with single values for their attributes do not
   generate variants.

Filter catalog by attributes
============================

Customers **can** filter the **catalog** based on product attributes and values, allowing them to
filter the catalog based only on the attribute(s) of their choice.

To enable **attributes filtering**, go to :menuselection:`Edit --> Customize` from your **main shop
page** and click on one of the **categories** in the left column. Here, enable either
:guilabel:`Left`, :guilabel:`Top`, or **both**, in the :guilabel:`Attributes` field.

> .. image:: variants/variants-catalog.png

Reviewer 2: ```suggestion
.. image:: variants/variants-catalog.png
```
Not really useful IMO - you don't really know what you are looking at.
If you really want a screenshot here, maybe show the categories option which is a bit difficult to find.
![image](https://user-images.githubusercontent.com/70884563/212948586-8a823938-1c5f-4946-97e3-c125a34c43b6.png)

   Pricelist formulas let you set advanced price computation methods for product variants. See
   :doc:`../maximizing_revenue/pricing`.
Filter catalog by attributes
============================

Disable/archive variants
========================
Customers **can** filter the **catalog** based on product attributes and values, allowing them to
filter the catalog based only on the attribute(s) of their choice.

You can disable/archive specific variants so they are no longer available in quotes or on your
website (not existing in your stock, deprecated, etc.). Simply select *Archive* in their detail
form. You can reactivate them the same way.
To enable **attributes filtering**, go to :menuselection:`Edit --> Customize` from your **main shop
page** and click on one of the **categories** in the left column. Here, enable either
:guilabel:`Left`, :guilabel:`Top`, or **both**, in the :guilabel:`Attributes` field.

.. image:: variants/archive.png
.. image:: variants/variants-catalog.png
   :align: center
   :alt: ecommerce product variants archive

To retrieve such items, hit *Archived* in the *Filters* category while searching the variants list.
   :alt: Variants catalog filtering

> .. image:: variants/archive-search.png

Reviewer 2: Not useful, especially without much context
